src/main/kotlin/no/nav/familie/ba/mottak/søknad/SøknadController.kt

```diff
@@ -24,7 +26,11 @@ class SøknadController(val featureToggleService: FeatureToggleService) {
         log.info("Lagring av søknad = $lagreSøknad")
 
         return if (lagreSøknad) {
-            ResponseEntity.ok(Ressurs.success(Kvittering("Søknad er mottatt", LocalDateTime.now())))
+            return try {
+                søknadService.motta(søknad)
+                ResponseEntity.ok(Ressurs.success(Kvittering("Søknad er mottatt", LocalDateTime.now())))
```

Comment:
burde denne kanskje bli flyttet inn i `motta`? Så kan vi bruke `opprettetTid` fra dbSøknad istedet for LocalDateTime.now(). Ser at ef har gjort det på den måten